plugins/admin/plugin/languages/en.yml

```diff
@@ -1,9 +1,9 @@
 en:
   command:
     ban:
-      description: Ban this sucker!
+      description: Prevents a player from joining the server.
```

Comment:
The "Ban this sucker" phrase is by design.

Reply:
reverted.